.github/CONTRIBUTING.md

```diff
@@ -49,18 +49,30 @@ Please see the [Contributor Guide in `PowerShell/PowerShell-Docs`](https://githu
 
 #### Spellchecking documentation
 
-Documentation are spellchecked. We make use of the
+Documentation is spellchecked. We use the
```

Comment:
This file apparently isn't being checked for markdown syntax issues.  
You can add it by updating this list https://github.com/PowerShell/PowerShell/blob/03cd3a2548a8029dc44b7c1155c992ea9d5f960b/test/common/markdown/markdown.tests.ps1#L57-L67

Not a requirement.  If you are using, vscode, you can just install the markdownlinq extension and you should get the issues as well.

Reply:
Okay 👍